File: velolabs_repos/lattis_android/app/src/main/java/com/lattis/ellipse/data/network/model/response/bike/ReserveBikeDataResponse.java
package com.lattis.ellipse.data.network.model.response.bike;

import com.google.gson.annotations.SerializedName;

/**
 * Created by ssd3 on 4/3/17.
 */

public class ReserveBikeDataResponse {

    @SerializedName("booked_on")
    private long booked_on;
    @SerializedName("expires_in")
    private int expires_in;
    @SerializedName("on_call_operator")
    private String on_call_operator;

    public String getOn_call_operator() {
        return on_call_operator;
    }

    public void setOn_call_operator(String on_call_operator) {
        this.on_call_operator = on_call_operator;
    }

    public long getBooked_on() {
        return booked_on;
    }

    public void setBooked_on(long booked_on) {
        this.booked_on = booked_on;
    }

    public int getExpires_in() {
        return expires_in;
    }

    public void setExpires_in(int expires_in) {
        this.expires_in = expires_in;
    }



}

File: velolabs_repos/lattis_android_v2/data/src/main/java/com/lattis/data/entity/response/bike/ReserveBikeDataResponse.kt
package com.lattis.data.entity.response.bike

import com.google.gson.annotations.SerializedName

/**
 * Created by ssd3 on 4/3/17.
 */

class ReserveBikeDataResponse {

    @SerializedName("booked_on")
    var booked_on: Long = 0
    @SerializedName("expires_in")
    var expires_in: Int = 0
    @SerializedName("on_call_operator")
    var on_call_operator: String? = null

    @SerializedName("booking_id")
    var booking_id: Int? = null


    @SerializedName("bike_id")
    var bike_id: Int? = null
    @SerializedName("port_id")
    var port_id: Int? = null
    @SerializedName("hub_id")
    var hub_id: Int? = null

    @SerializedName("device_type")
    var device_type: String? = null

    @SerializedName("trip_id")
    var trip_id: Int? = null

    @SerializedName("do_not_track_trip")
    var do_not_track_trip: String? = null


}
